.github/ISSUE_TEMPLATE.md

```diff
@@ -7,9 +7,17 @@
 ## What happens:
 1.
 
-## Logs
-- Logs
+## Traceback
+[//]: # (If there is a traceback please share it in a quote! You can do this by pasting the traceback text, then highlight it and press the quote button.)
 
-## Extra info:
-- Tested on
+## SDK version and environment
+- Tested on [0.12.4](https://github.com/bunq/sdk_java/releases/tag/0.12.4)
```

Comment:
To save yourself some work, you might want to add a script in the future that automatically updates these numbers to the newest version. (see also [youtube-dl](https://github.com/rg3/youtube-dl/blob/master/.github/ISSUE_TEMPLATE_tmpl.md) as an example).

Reply:
These version number will be bumped during version bump at during the release phase

Reply:
youtube-dl also bumps the version number on a new release, but they have automated it. It's easy to forget one when the numbers appear in more places/examples. No biggy tho for now.

Reply:
Ah I see what you mean 😁 Will add this to my todo list! 😏